packaging/kryoptic.spec

```diff
@@ -0,0 +1,69 @@
+# Generated by rust2rpm 26
+%bcond_without check
+
+# prevent library files from being installed -- not on cargo.io !
+%global cargo_install_lib 0
+
+%global revision 3b7a9137ddee914938bd02489150844aaa909125
+%global short_revision 3b7a9137
+%global revision_date 20240916
+
+Name:           kryoptic
+Version:        0.1.0^%{revision_date}.git%{short_revision}
+Release:        %autorelease
+Summary:        PKCS #11 software token written in Rust
+
+SourceLicense:  GPL-3.0
+# Apache-2.0
+# Apache-2.0 OR BSL-1.0
+# Apache-2.0 OR MIT
+# BSD-2-Clause OR Apache-2.0 OR MIT
+# BSD-3-Clause
+# MIT
+# MIT OR Apache-2.0
+# MIT-0 OR Apache-2.0
+# Unlicense OR MIT
+License: Apache-2.0 AND (Apache-2.0 OR BSL-1.0) AND (Apache-2.0 OR MIT) AND (BSD-2-Clause OR Apache-2.0 OR MIT) AND (BSD-3-Clause) AND (MIT) AND (MIT OR Apache-2.0) AND (MIT-0 OR Apache-2.0) AND (Unlicense OR MIT)
+# LICENSE.dependencies contains a full license breakdown
+
+URL:            https://github.com/latchset/kryoptic
+Source:         https://github.com/latchset/kryoptic/archive/%{revision}.zip
+
+BuildRequires:  cargo-rpm-macros >= 26
+BuildRequires:  openssl-devel
+
+%global _description %{expand:
+A PKCS #11 software token written in Rust.}
+
+%description %{_description}
+
+%prep
+%autosetup -n %{name}-%{revision} -p1
+%cargo_prep
+
+%generate_buildrequires
+%cargo_generate_buildrequires -n -f dynamic
+
+%build
+%cargo_build -n -f dynamic
+%{cargo_license_summary -n -f dynamic}
+%{cargo_license -n -f dynamic} > LICENSE.dependencies
+
+%install
+%cargo_install -n -f dynamic
+# FIXME: install shared library
```

Comment:
Is this a FIXME that should be kept ?

Reply:
Good catch! I had it in one of previous iterations, but it got lost. Will fix.